subsys/bluetooth/services/cgms/cgms_internal.h

```diff
@@ -0,0 +1,141 @@
+#define BT_SERVICE_CGMS_FEAT_SENSOR_MALFUNCTION_DETECTION_SUPPORTED          (0x01 << 6)
+#define BT_SERVICE_CGMS_FEAT_SENSOR_TEMPERATURE_HIGH_LOW_DETECTION_SUPPORTED (0x01 << 7)
+#define BT_SERVICE_CGMS_FEAT_SENSOR_RESULT_HIGH_LOW_DETECTION_SUPPORTED      (0x01 << 8)
+#define BT_SERVICE_CGMS_FEAT_LOW_BATTERY_DETECTION_SUPPORTED                 (0x01 << 9)
+#define BT_SERVICE_CGMS_FEAT_SENSOR_TYPE_ERROR_DETECTION_SUPPORTED           (0x01 << 10)
+#define BT_SERVICE_CGMS_FEAT_GENERAL_DEVICE_FAULT_SUPPORTED                  (0x01 << 11)
+#define BT_SERVICE_CGMS_FEAT_E2E_CRC_SUPPORTED                               (0x01 << 12)
+#define BT_SERVICE_CGMS_FEAT_MULTIPLE_BOND_SUPPORTED                         (0x01 << 13)
+#define BT_SERVICE_CGMS_FEAT_MULTIPLE_SESSIONS_SUPPORTED                     (0x01 << 14)
+#define BT_SERVICE_CGMS_FEAT_CGM_TREND_INFORMATION_SUPPORTED                 (0x01 << 15)
+#define BT_SERVICE_CGMS_FEAT_CGM_QUALITY_SUPPORTED                           (0x01 << 16)
+
+/* Continuous Glucose Monitoring type */
+#define BT_SERVICE_CGMS_MEAS_TYPE_CAP_BLOOD    0x01
+#define BT_SERVICE_CGMS_MEAS_TYPE_CAP_PLASMA   0x02
+#define BT_SERVICE_CGMS_MEAS_TYPE_VEN_BLOOD    0x03
+#define BT_SERVICE_CGMS_MEAS_TYPE_VEN_PLASMA   0x04
+#define BT_SERVICE_CGMS_MEAS_TYPE_ART_BLOOD    0x05
+#define BT_SERVICE_CGMS_MEAS_TYPE_ART_PLASMA   0x06
+#define BT_SERVICE_CGMS_MEAS_TYPE_UNDET_BLOOD  0x07
+#define BT_SERVICE_CGMS_MEAS_TYPE_UNDET_PLASMA 0x08
+#define BT_SERVICE_CGMS_MEAS_TYPE_FLUID        0x09
+#define BT_SERVICE_CGMS_MEAS_TYPE_CONTROL      0x0A
+
+/* CGM sample location */
+#define BT_SERVICE_CGMS_MEAS_LOC_FINGER     0x01
+#define BT_SERVICE_CGMS_MEAS_LOC_AST        0x02
+#define BT_SERVICE_CGMS_MEAS_LOC_EAR        0x03
+#define BT_SERVICE_CGMS_MEAS_LOC_CONTROL    0x04
+#define BT_SERVICE_CGMS_MEAS_LOC_SUB_TISSUE 0x05
+#define BT_SERVICE_CGMS_MEAS_LOC_NOT_AVAIL  0x0F
+
+/* CGM Measurement Sensor Status Annunciation */
+#define BT_SERVICE_CGMS_STATUS_SESSION_STOPPED                  (0x01 << 0)
+#define BT_SERVICE_CGMS_STATUS_DEVICE_BATTERY_LOW               (0x01 << 1)
+#define BT_SERVICE_CGMS_STATUS_SENSOR_TYPE_INCORRECT_FOR_DEVICE (0x01 << 2)
+#define BT_SERVICE_CGMS_STATUS_SENSOR_MALFUNCTION               (0x01 << 3)
+#define BT_SERVICE_CGMS_STATUS_DEVICE_SPECIFIC_ALERT            (0x01 << 4)
+#define BT_SERVICE_CGMS_STATUS_GENERAL_DEVICE_FAULT             (0x01 << 5)
+
+/* CGM Measurement flags */
+#define BT_SERVICE_CGMS_FLAG_TREND_INFO_PRESENT          0x01
+#define BT_SERVICE_CGMS_FLAGS_QUALITY_PRESENT            0x02
+#define BT_SERVICE_CGMS_STATUS_FLAGS_WARNING_OCT_PRESENT 0x20
+#define BT_SERVICE_CGMS_STATUS_FLAGS_CALTEMP_OCT_PRESENT 0x40
+#define BT_SERVICE_CGMS_STATUS_FLAGS_STATUS_OCT_PRESENT  0x80
+
+struct bt_cgm_sensor_annunc_t {
+	/**< Warning annunciation. */
+	uint8_t	warning;
+	/**< Calibration and Temperature annunciation. */
+	uint8_t calib_temp;
+	/**< Status annunciation. */
```

Comment:
```suggestion
	/** Status annunciation. */
```
And in other places. 
See: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#structs